Hunting Queries/McAfeeePO/McAfeeEPOMultipleThreats.yaml

```diff
@@ -0,0 +1,26 @@
+id: 1ef23489-a840-4b43-b33d-a921da24c85c
+name: McAfee ePO - Sources with multiple threats
+description: |
+  'Query searches for sources with several different threats.'
+requiredDataConnectors:
+  - connectorId: McAfeeePO
+    dataTypes:
+      - Syslog
+tactics:
+  - InitialAccess
+query: |
+  let lbtime = 24h;
+  let threshold = 1;
+  McAfeeEPOEvent
+  | where TimeGenerated > ago(lbtime)
+  | where isnotempty(ThreatName)
+  | where ThreatName != '_'
+  | summarize th_cnt = makeset(ThreatName) by DvcIpAddr
+  | where array_len(ThreatList) > threshold

```

Comment:
You have no `ThreatList` field coming thru after the summarize here.  I assume you meant to use `th_cnt`

Reply:
fixed